tests/unit_tests/gui/ertwidgets/test_search_box.py

```diff
@@ -0,0 +1,52 @@
+from qtpy.QtGui import QColor, QTextCursor
+from qtpy.QtWidgets import QApplication, QPlainTextEdit
+
+from ert.gui.tools.search_bar import SearchBar
+
+
+def test_search_box_highlights_matching_text():
```

Comment:
Just wondering if all these tests could be tackled with one parametrize?

Reply:
Sure I can combine them in one test

Reply:
Done